src/enums/button-selector.ts

```diff
@@ -0,0 +1,5 @@
+export enum ButtonOPT {

```

Comment:
What's the meaning of the name `ButtonOPT`?

Reply:
export button option for every RP can choose to use "display of button".

Reply:
So, maybe you could use naming that more descriptive and informative like `ButtonOption` or `ButtonText`, or you can also simply use `if..else` or ternary operator within the component na krub. What do you think?

Reply:
I could change the name of the component, But for using "if else" or "using ternary operator" I'm still not familiar with HTML.sory krub but will try krub.